src/Enable.Extensions.Queuing.RabbitMQ/Internal/RabbitMQQueueClient.cs

```diff
@@ -240,5 +204,61 @@ private string GetDeadLetterQueueName(string queueName)
         {
             return $"{queueName}.dead-letter";
         }
+
+        private void ConfigureBasicQos(int prefetchCount)
+        {
+            lock (_channel)
+            {
+                // Here we are assuming a "worker queue", where multiple consumers are
+                // competing to draw from a single queue in order to spread workload
+                // across the consumers.
+                _channel.BasicQos(
+                    prefetchSize: 0,
+                    prefetchCount: Convert.ToUInt16(prefetchCount),
+                    global: false);
+            }
+        }
+
+        private async Task OnReceived(
```

Comment:
Making this an `async` method allows us to `await` the calls to other `async` methods within the method body.